.github/workflows/build.yml

```
@@ -71,17 +68,17 @@ jobs:
         ndkversion: [21]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Set up JDK 1.11
+      - name: Set up JDK 17
```

```suggestion
      - name: Set up JDK
```

Why bother? I don't think it's worth duplicating.

---

Kept original style, but maybe w/o version is cleaner. Thanks